docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '3.5'
+services:
+  api:
+    container_name: shopify_api
+    image: ruby:2.4.6
```

Comment:
Is it the correct ruby version?

Reply:
Last update: https://github.com/Shopify/shopify_api/commit/57c24ed888e1acff0c0a1aa0057a5a9754cadc7b